--- nixos/modules/profiles/base.nix.orig
+++ nixos/modules/profiles/base.nix
@@ -49,7 +49,7 @@
   ];

   # Include support for various filesystems.
-  boot.supportedFilesystems = [ "btrfs" "reiserfs" "vfat" "f2fs" "xfs" "zfs" "ntfs" "cifs" ];
+  boot.supportedFilesystems = [ "btrfs" "reiserfs" "vfat" "f2fs" "xfs" "ntfs" "cifs" ];

   # Configure host id for ZFS to work
   networking.hostId = lib.mkDefault "8425e349";
